conjur/logic/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""
+logic package
+
+This package contain the logic classes
+"""
+
+from conjur.logic.credentials_from_file import CredentialsFromFile
```

Comment:
why is this part of logic?

Reply:
This is not a command logic file but it preforms the logic of getting configuration from netrc file. we talked a week ago about this